singlestoredb_cloud/CHANGELOG.md

```diff
@@ -1,7 +1,13 @@
 # CHANGELOG - SingleStore DB Cloud
 
+## 1.0.1 / 2023-10-16
+
+***Changed***: 
+
+* Updated dashboard for better experience
+
 ## 1.0.0 / 2023-07-10
 
-***Added***:
+***Added***: 
```

Comment:
There is a space after the word ***Added*** here and there previously was not. Could this please be removed. Thanks!